lib/phctl.rb

```diff
@@ -109,6 +109,15 @@ def costreport
       run_common_job(Reports::CostReport, options)
     end
 
+    desc "costreport-workflow --ht-item-count NUM --ht-item-pd-count NUM (--chunk-size SIZE)", "Dump records from solr, split into chunks of chunk-size records, generate frequency tables for each chunk, sum the resulting frequency tables, and generate a cost report based on that table."
```

Comment:
Just noticing that the output from `bash phctl.sh report` is starting to get hard to read with such long invocations, but I reckon that's just life.